ksql-engine/src/test/java/io/confluent/ksql/util/KafkaTopicClientTest.java

```diff
@@ -0,0 +1,166 @@
+    Map<ConfigResource, Config> config = new HashMap<>();
+    config.put(new ConfigResource(ConfigResource.Type.BROKER, "1"), new Config
+        (Collections.singletonList(configEntry)));
+    expect(describeConfigsResult.all()).andReturn(KafkaFuture.completedFuture(config));
+    ListTopicsResult listTopicsResult = mock(ListTopicsResult.class);
+    List<String> topicNamesList = Arrays.asList(topicName1, topicName2, topicName3);
+    expect(listTopicsResult.names()).andReturn(KafkaFuture.completedFuture(new HashSet<>
+                                                                               (topicNamesList)));
+
+    DeleteTopicsResult deleteTopicsResult = mock(DeleteTopicsResult.class);
+    expect(deleteTopicsResult.values()).andReturn(Collections.singletonMap(topicName1, KafkaFuture
+        .allOf()));
+
+
+    CreateTopicsResult createTopicsResult = mock(CreateTopicsResult.class);
+    KafkaFutureImpl<Optional<Void>> voidKafkaFuture = new KafkaFutureImpl<Optional<Void>>();
+    expect(createTopicsResult.all()).andReturn(KafkaFuture.allOf());
+
+    TopicPartitionInfo topicPartitionInfo = new TopicPartitionInfo(0, node, Collections
+        .singletonList(node), Collections.singletonList(node));
+    TopicDescription topicDescription = new TopicDescription(topicName1, false,
+                                                             Collections.singletonList
+                                                                 (topicPartitionInfo));
+    DescribeTopicsResult describeTopicsResult = mock(DescribeTopicsResult.class);
+    expect(describeTopicsResult.all()).andReturn(KafkaFuture.completedFuture(Collections
+                                                                                 .singletonMap
+                                                                                     (topicName1,
+                                                                                      topicDescription
+                                                                                      )));
+
+    adminClient = mock(AdminClient.class);
+    expect(adminClient.describeCluster()).andReturn(describeClusterResult);
+    expect(adminClient.createTopics(anyObject())).andReturn(createTopicsResult);
+    expect(adminClient.deleteTopics(anyObject())).andReturn(deleteTopicsResult);
+    expect(adminClient.describeConfigs(anyObject())).andReturn(describeConfigsResult);
+    expect(adminClient.listTopics()).andReturn(listTopicsResult);
+    expect(adminClient.describeTopics(anyObject())).andReturn(describeTopicsResult);
+
+    replay(describeClusterResult);
+    replay(describeConfigsResult);
+    replay(describeTopicsResult);
+    replay(deleteTopicsResult);
+    replay(createTopicsResult);
+    replay(listTopicsResult);
+    replay(adminClient);
+    kafkaTopicClient = new KafkaTopicClientImpl(adminClient);
+  }
+
+  @Test
+  public void testCreateTopic() {
+    kafkaTopicClient.createTopic("test", 1, (short)1);
```

Comment:
`EasyMock.verify(...)` - same elsewhere - you should verify the important interactions, not necessarily all, but in this case, that AdminClient